MapboxSpeech.framework/Headers/MapboxSpeech-Swift.h

```diff
@@ -0,0 +1,251 @@
+// Generated by Apple Swift version 4.0.3 (swiftlang-900.0.74.1 clang-900.0.39.2)
```

Comment:
Did you mean to check in the framework bundle?

Reply:
Aw, my mistake.